src/containers/Home/Jobs/Pagination/index.tsx

```diff
@@ -1,10 +1,15 @@
 import { useEffect, useState } from "react"
+import { useSelector } from "react-redux";
+import { RootState } from "../../../Redux/store/store";
 
-export default function Pagination(props: { jobsPerPage: number, totalJobs: number, paginate: any, currentPage: number }) {
+export default function Pagination(props: { jobsPerPage: number, totalJobs: number, paginate: any, currentPage: number,totalSearchJob: number }) {
 
-    const { jobsPerPage, totalJobs, paginate, currentPage } = props
+    const { jobsPerPage, totalJobs, paginate, currentPage, totalSearchJob } = props
     const [pageNumber, setPageNumber] = useState<Number[]>([]);
-    const totalPage = Math.ceil(totalJobs / jobsPerPage);
+    const search = useSelector((state:RootState)=> state.search)
```

Comment:
should `{ search } = useSelector(...)` for shorten

Reply:
i've changed